frontend/src/pages/ProcessDataPage/ProcessedDataPage.tsx

```diff
@@ -116,10 +169,12 @@ const ProcessedDataPage = function () {
      */
     const getRendersOfFiles = () => {
         console.assert(files.length > 0, "Files array should contain data for rendering");
-        renders = files.map((file: ProcessedFileData) => {
+        renders = files.map((file: any) => {
```

Comment:
For this we can define at the top level a type just to avoid an any type.

Can do something like this

```
// Inherits properties of ProcessedFileData and adds a new watch property as well
type ProcessedFile = ProcessedFileData & { 
    watch: DataType,
};
```

Then you would be able to do this

```suggestion
        renders = files.map((file: ProcessedFile) => {
```